helm/inji-config-server/values.yaml

```diff
@@ -103,6 +103,18 @@ envVariables:
   - name: SPRING_CLOUD_CONFIG_SERVER_OVERRIDES_MOSIP_ESIGNET_INSURANCE_HOST
     valueFrom:
       configMapKeyRef:
-         key: mosip-esignet-insurance-host
-         name: global
+        key: mosip-esignet-insurance-host
```

Comment:
IS this needed permanently in config values?
As this is just needed for one plugin.

Reply:
Yes chandra, it is needed for mimoto, to OVP flow and OIVCI flows